src/applications/representative-form-upload/helpers/intent-to-file-helper.jsx

```
@@ -0,0 +1,87 @@
+import { isEmpty } from 'lodash';
+import { apiRequest } from 'platform/utilities/api';
+import environment from '@department-of-veterans-affairs/platform-utilities/environment';
+
+const goPathAfterGettingITF = (
+  intent,
+  formData,
+  goPath,
+  goNextPath,
+  setFormData,
+  urlPrefix,
+) => {
+  const formDataToSet = { ...formData, 'view:activeITF': intent };
+
+  setFormData(formDataToSet);
+
+  if (!isEmpty(formDataToSet?.['view:activeITF'])) {
+    goPath(`${urlPrefix}existing-itf`);
+  } else {
+    goNextPath();
+  }
+};
+
+const fetchIntentToFile = async (
+  formData,
+  benefitType,
+  urlPrefix,
+  goPath,
+  goNextPath,
+) => {
+  let params = `?veteranFirstName=${formData.veteranFullName.first}`;
+  params = `${params}&veteranLastName=${formData.veteranFullName.last}`;
```

Can we URL-encode the ITF query parameters and surface an error when the lookup fails, rather than silently calling goNextPath()?

---

I'll wrap these in `encodeURIComponent` in a follow-on PR.